app/assets/javascripts/no_sunet.js

```diff
@@ -0,0 +1,43 @@
+$(document).on('click', '#no-sunet-button', function(){
+  setTimeout(function(){
+      $('#request_user_attributes_library_id').focus();
+  }, 1);
+
+  $('form').on('keypress', function(e) {
+    if(e.which == 13) {
+      $('.form-group input').checkForm(e);
```

Comment:
Sorry, I misspoke. Yes, that returns all the `inputs` in `.form-group` elements, but why are we dealing with them unconditionally?  In `checkForm` below you loop through all the matching elements and check if they have been filled out.  What about if there is an input that isn't required and does not need to be filled out?  Would this code add the tooltip if a non-required input was left blank?